src/main/resources/db/migration/V1__init.sql

```diff
@@ -35,12 +35,24 @@ CREATE TABLE contacts (
     ON DELETE CASCADE
 );
 
+CREATE TABLE approvements (
+  task_id      INT,
+  user_id      INT,
+  mark         ENUM ('GOOD', 'BAD', 'VIEWED') DEFAULT 'VIEWED',
+  approve_time TIMESTAMP NOT NULL,
+  FOREIGN KEY (task_id) REFERENCES tasks (id)
+    ON DELETE CASCADE,
+  FOREIGN KEY (user_id) REFERENCES users (id)
```

Comment:
I think it is good to save opinion of people, who can't approve anymore because he had gone to other project, but he is very компитентен in this question